src/test/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/entity/GameBeanValidationTest.java

```diff
@@ -21,27 +29,68 @@ void setup() {
     validatorFactory.close();
   }
 
-  @Test
-  void valid_player_is_accepted() {
+  @ParameterizedTest(name = "{0}")
+  @MethodSource("gameNames")
+  void validates_game_name(String name, ValidationListMatcher validationListMatcher) {
     Game game = new Game();
-    game.setName("test");
+    if (name != null) {
+      game.setName(name);
```

Comment:
Why do we jump over when the value of name is null?

Reply:
Because if the name is null, game.SetName throws IllegalArgumentException